ui/app/serializers/pki/certificate/base.js

```diff
@@ -0,0 +1,30 @@
+import { parseCertificate } from 'vault/helpers/parse-pki-cert';
+import ApplicationSerializer from '../../application';
+
+export default class PkiCertificateBaseSerializer extends ApplicationSerializer {
+  primaryKey = 'serial_number';
+
+  attrs = {
+    role: { serialize: false },
+  };
+
+  normalizeResponse(store, primaryModelClass, payload, id, requestType) {
+    if (payload.data.certificate) {
+      // Parse certificate back from the API and add to payload
+      const parsedCert = parseCertificate(payload.data.certificate);
+      // convert issueDate to same format as other date values
+      if (parsedCert.issue_date) {
+        parsedCert.issue_date = parsedCert.issue_date.valueOf();
```

Comment:
What do you think about adding `valueOf()`to `issue_date` directly in `helpers/parse-pki-cert.js` instead? That's how we handle it for both `not_valid_after` and `not_valid_before`

Then we could remove the conditional here

Reply:
I originally had it that way but then realized that the old `cert` serializer and `pki-ca-certificate` adapter are also using the helper and I didn't want to break anything in the existing implementation.

I just double checked and there is an `Invalid time value` error thrown on the pki/show/cert route if I make the update in the helper. I think rather than tracking down where it is all used it would be best to leave a comment above the conditional that it can be moved to the helper once the old pki is removed.